src/Calculator/Views/HistoryList.xaml

```diff
@@ -50,6 +50,10 @@
             </muxc:SwipeItems>
 
             <MenuFlyout x:Name="HistoryContextMenu">
+                <MenuFlyoutItem x:Name="CopyHistoryMenuItem"
```

Comment:
No need for `x:Name` since we won't be programatically accessing this item with the change in localization.